src/components/shared/NavBar.jsx

```diff
@@ -0,0 +1,84 @@
+import { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import Text from './typography/Text';
+import Navigation from '../../config/Navigation';
+import menuIcon from '../../assets/images/menu.png';
+import unionIcon from '../../assets/images/Union.png';
+import image from '../../assets/images/image.png';
+
+function NavBar() {
+	const { navItems } = Navigation;
+	const [isMobile, setIsMobile] = useState(false);
+	const [isNavOpen, setIsNavOpen] = useState(false);
+	const [menuIconSrc, setMenuIconSrc] = useState(menuIcon);
+
+	useEffect(() => {
+		const handleResize = () => {
+			setIsMobile(window.innerWidth < 1185);
+		};
+		window.addEventListener('resize', handleResize);
+		handleResize();
+		return () => window.removeEventListener('resize', handleResize);
+	}, []);
+
+	const toggleNav = () => {
+		if (isMobile) {
+			setIsNavOpen(!isNavOpen);
+			setMenuIconSrc(isNavOpen ? menuIcon : unionIcon);
+		}
+	};
+
+	return (
+		<div>
+			<div
+				className='navbar flex justify-between items-center opacity-53 bg-hard-light bg-center bg-cover h-[73.8px] overflow-hidden'
+				style={{ backgroundImage: `url(${image})` }}>
+				<div className='ml-[56.6px]'>Logo</div>
```

Comment:
Try use padding in parent div. Furthermore where is the responsiveness as padding is too much for mobile screens.